build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'net.minecraftforge.gradle:ForgeGradle:4.1.+'
+        classpath 'net.minecraftforge.gradle:ForgeGradle:5.+'
```

Comment:
With how breaking changes get handled for FG, I feel like it may almost make sense to just hold off on this slightly longer until this can be 5.1.+